### DISCUSSION TITLE
RFID Cards reset

### DISCUSSION THREAD

**Discussion by fwenzelb** [MiczFlor/RPi-Jukebox-RFID#1439] [Q&A]
Hi guys. Is it possible to reset a RFID-Card? I configured one card to be "pause" and now i want it to use to play a folder. I tried to change the configuration through the webinterface but the card keeps beeing configured as "pause". Can anyone help?

**Comment by MiczFlor** [comment]
Hi @fwenzelb 
at the moment, there is no "catching" of duplicates (now that I am writing this, I remember having written some code for this about a year ago... for the web interface... I'll have to have a look).
You have to manually edit the file `settings/rfid_trigger_play.conf` on the Phoniebox and *reset* the pause ID by changing the matching line to:

`CMDPAUSE="%CMDPAUSE%"`

See this sample file for reference:
https://github.com/MiczFlor/RPi-Jukebox-RFID/blob/develop/settings/rfid_trigger_play.conf.sample

**Comment by steveleader** [comment]
You can download the current configuration of RFID cards as csv in the web interface. Then you can delete the respective line in the csv and upload the csv again using the webinterface with the option to only use this new list without keeping the existing configuration. 
For me that worked well to unbind a specific card.
